pkg/logql/metrics.go

```diff
@@ -108,11 +109,12 @@ func RecordRangeAndInstantQueryMetrics(
 
 	queryTags, _ := ctx.Value(httpreq.QueryTagsHTTPHeader).(string) // it's ok to be empty.
 
-	logValues := make([]interface{}, 0, 20)
+	logValues := make([]interface{}, 0, 21)
```

Comment:
By my count, there are `30` items

Reply:
Haha your math is better than mine. Fixed!